Expired/Views/SettingView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SettingView.swift
+//  Expired
+//
+//  Created by Karl Michael Reyes on 2023-02-06.
+//
+
+import SwiftUI
+
+struct SettingView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+    @EnvironmentObject var productStore: ProductStore
+    @State private var showingDeleteAlert: Bool = false
+    @State private var showingSuccessAlert: Bool = false
+    var body: some View {
+        // Input here for Sandeep's part
+        List {
+            Section(header: Text("Archive List")) {
+                Text("test")
+            }
+
```

Comment:
```suggestion
        List {
```

- It's a better idea to keep a feature branch only changes what it's supposed to change.